### ISSUE TITLE
  Sauna stuck bug

### ISSUE THREAD

**Issue by vkeldh** [Uuvana-Studios/longvinter-windows-client#1202]
**Describe the bug**

A bug occurs where one sauna is waiting at a suitable location, and if a person enters after that, both people cannot move

**To Reproduce**
1. One person enters the sauna and waits on the right near the entrance.
2. When one more person enters, there is a bug where it cannot move

**Desktop (please complete the following information):**
 - OS: Windows
 - Game Version 1.0.8b

https://youtu.be/GvYEU6bA3io

**Comment by faketucker** [comment]
Duplicate of #1138
